flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinator.java

```diff
@@ -36,7 +37,7 @@
  * </ol>
  */
 @Internal
-interface SubtaskCheckpointCoordinator {
+interface SubtaskCheckpointCoordinator extends Closeable {
```

Comment:
It seems the `close` is never called by `StreamTask`, and this change should be moved into the commit when `SubtaskCheckpointCoordinator` is introduced.